test/Microsoft.AspNetCore.Mvc.Razor.Test/DependencyInjection/MvcRazorMvcBuilderExtensionsTest.cs

```diff
@@ -36,8 +36,9 @@ public void AddTagHelpersAsServices_ReplacesTagHelperActivatorAndTagHelperTypeRe
             var activatorDescriptor = Assert.Single(services.ToList(), d => d.ServiceType == typeof(ITagHelperActivator));
             Assert.Equal(typeof(ServiceBasedTagHelperActivator), activatorDescriptor.ImplementationType);
 
-            var resolverDescriptor = Assert.Single(services.ToList(), d => d.ServiceType == typeof(ITagHelperTypeResolver));
-            Assert.Equal(typeof(FeatureTagHelperTypeResolver), resolverDescriptor.ImplementationType);
+            // This was removed to facilitate the move to new razor and will be re-added as part of https://github.com/aspnet/Mvc/issues/5768
+            //var resolverDescriptor = Assert.Single(services.ToList(), d => d.ServiceType == typeof(ITagHelperTypeResolver));
```

Comment:
Remove it for now. It's in the source control history if you need to look it up. Let's not leave commented out code.